src/python/bot/fuzzers/honggfuzz/engine.py

```diff
@@ -55,6 +55,7 @@ def _get_runner():
   if not os.path.exists(honggfuzz_path):
     raise HonggfuzzError('honggfuzz not found in build')
 
+  os.chmod(honggfuzz_path, 0o755)
```

Comment:
If we are doing this here, why doing in prepare and reproduce ?

Reply:
That's the target binary, while this is the honggfuzz driver binary.